.github/workflows/deploy_to_aws.yml

```
@@ -228,6 +229,14 @@ jobs:
           push: true
           tags: ${{ vars.ECR_REGISTRY }}/${{ vars.IMAGE_NAME }}-database-anonymizer-${{ steps.get_env_name.outputs.lowercase }}:${{ env.GIT_COMMIT }}
 
+      - name: Build and push Anonymize Database Dump Docker image to ECR for ${{ vars.ENV_NAME }}
+        uses: docker/build-push-action@v2
```

actionlint

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)